adaptdl/adaptdl/torch/__init__.py

```diff
@@ -48,8 +60,18 @@ def init_process_group(backend):
                 break
         response.raise_for_status()
         master_addr = response.json()[0]
+        sched_version = os.environ.get("ADAPTDL_VERSION")
```

Comment:
Rename env var to `ADAPTDL_SCHED_VERSION` to make clear this is the scheduler's version.

Reply:
Updated